packages/typeorm/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@kuai/typeorm",
+  "version": "0.0.1",
+  "description": "Inject TypeOrm by InversifyJs",
+  "author": "felicityin <yinjingping2022@gmail.com>",
+  "homepage": "https://github.com/ckb-js/kuai#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ckb-js/kuai.git"
+  },
+  "scripts": {
+    "build": "tsc",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "bugs": {
+    "url": "https://github.com/ckb-js/kuai/issues"
+  },
+  "dependencies": {
+    "reflect-metadata": "0.1.13",
+    "inversify": "^6.0.1",
+    "rxjs": "^7.5.7",
+    "typeorm": "0.3.10",
+    "tslib": "2.4.1"
+  },
+  "devDependencies": {
+    "@jest/globals": "29.3.1",
+    "typescript": "4.8.4",
+    "ts-node": "10.7.0",
+    "pg": "^8.8.0"
```

Comment:
Yes, it only used by examples. And examples will be added in the next pr. This library is universal, and there is no limit to which database users must use.

Reply:
> Yes, it only used by examples. And examples will be added in the next pr. This library is universal, and there is no limit to which database users must use.

We've got the conclusion that only typeorm and PostgreSQL are going to be supported in implementation for simplicity because PostgreSQL is widely used among our teams(about 2 weeks ago)

Reply:
Yes, our team will use PostgreSQL, but the library can support others.

Reply:
> Yes, our team will use PostgreSQL, but the library can support all.

The library supports all for various use cases while kuai specifies PG because we have an opinion on the database

Reply:
`pg` could be removed once we're going to support other databases

Reply:
> `pg` could be removed once we're going to support other databases

OK